src/consent-manager/container.tsx

```diff
@@ -41,7 +41,7 @@ interface ContainerProps {
   isConsentRequired: boolean
   implyConsentOnInteraction: boolean
   bannerContent: React.ReactNode
-  bannerSubContent: React.ReactNode
+  bannerSubContent: string | undefined
```

Comment:
I think it might make sense to have the type be `React.ReactNode | string`. That way you can apply your own styling to the sub-content by wrapping the contents in a `span` (or even include an `img` if you really wanted).